umbral/bignum.py

```diff
@@ -253,11 +254,16 @@ def __hash__(self):
 
 
 def hash_to_bn(crypto_items, params):
+    if not isinstance(crypto_items, list):
+        crypto_items = [crypto_items]
```

Comment:
We should be using argument packing here.

It would look like this:
`def hash_to_bn(params, *crypto_items)`

This packs all the `crypto_items` into a tuple that you can iterate over. You can call it like this:
`new_bn = hash_to_bn(params, x_1, x_2, x_3, x_4, ...)`

Then you don't have to worry about it being an iterable.

Reply:
Perhaps this can also be in a different PR, too?

Reply:
OK, I will revert this changes. Argument packing makes sense, I will do that in a different PR